cli/command/container/rm.go

```diff
@@ -27,9 +27,10 @@ func NewRmCommand(dockerCli command.Cli) *cobra.Command {
 	var opts rmOptions
 
 	cmd := &cobra.Command{
-		Use:   "rm [OPTIONS] CONTAINER [CONTAINER...]",
-		Short: "Remove one or more containers",
-		Args:  cli.RequiresMinArgs(1),
+		Use:     "rm [OPTIONS] CONTAINER [CONTAINER...]",
+		Aliases: []string{"remove"},
```

Comment:
Can you add it to the `Annotations` below as well? https://github.com/docker/cli/blob/645395cc779a04bbf9dc6027bbf4d9b5016d65ec/cli/command/container/rm.go#L37-L39

(see https://github.com/docker/cli/pull/3694 why we added those)

Reply:
done